terragrunt/tools/grafana/configs/entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+sed -i "s/GRAFANA_CLOUDWATCH_ACCOUNT_ID/${GRAFANA_CLOUDWATCH_ACCOUNT_ID}/g" /etc/grafana/provisioning/dashboards/application/*
```

Comment:
I assume it isn't possible to use env vars in config files?

Reply:
Correct assumption, Grafana's dashboards are plain JSON. so here a simple `sed` does the rendering of the env_var into them